variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "env_code" {
+  type = string
+}
+
+variable vpc_cidr {}
+
+variable  private_cidr {}
+
+variable public_cidr {}
```

Comment:
Please add a newline to the end of this file

Reply:
newline has been appended to variables.tf file